parser.go

```diff
@@ -329,28 +328,14 @@ func (p *parser) readPythonMultilines(line string, bufferSize int) (string, erro
 			return line, nil
 		}
 
-		// Determine indent size and line prefix.
-		currentIndentSize := len(peekMatches[1])
-		if indentSize < 1 {
-			indentSize = currentIndentSize
-			p.debug("readPythonMultilines: indent size is %d", indentSize)
-		}
-
-		// Make sure each line is indented at least as far as first line.
-		if currentIndentSize < indentSize {
-			p.debug("readPythonMultilines: end of value, current indent: %d, expected indent: %d, line: %q", currentIndentSize, indentSize, line)
-			return line, nil
-		}
-
```

Comment:
What is the rationale of deleting these lines? Without these, it seems we can't determine indent size for the multiline value?

Reply:
The main idea here is to not bypass any miss indentation from the source which means we're looking for **break lines** and not for "malformed" multiline values. Having it in the logic makes the first line to always determine the other's indentation.

Reply:
> The main idea here is to not bypass any miss indentation from the source which means we're looking for **break lines** and not for "malformed" multiline values. Having it in the logic makes the first line to always determine the other's indentation.

What is definition of a "break line", is it a line that failed to be matched with `pythonMultiline.FindStringSubmatch`?

Reply:
All look good to me, just want to confirm the impact of deleting these lines.